### PR TITLE
refactored /app/pages/Profile/index.js to use OverMind and change it …

### PR DESCRIPTION
**Pull request by hardiked**
## What kind of change does this PR introduce? 
This PR refactor /app/pages/Profile/index.js to use OverMind and change it to tsx file.

## What steps did you take to test this? This is required before we can merge, make sure to test the flow you've updated.

1. ```yarn test``` successful
2. ```yarn lint``` is not successful but the error is not related to the scope of this PR
3. ```yarn start``` successful

## Checklist
- [x] Testing 
- [x] Ready to be merged

### CODE REVIEW

File: packages/app/config/stubs/lru-cache.js [hunk 1]
```diff
@@ -1,482 +0,0 @@
-'use strict';
```
Comment (CompuIves) [inline packages/app/config/stubs/lru-cache.js]: Why did you delete this? 🤔

File: packages/app/src/app/pages/Profile/index.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
+/* @flow */
```
Comment (christianalfoni) [inline packages/app/src/app/pages/Profile/index.tsx]:
```suggestion

```

File: packages/app/config/stubs/lru-cache.js (hunk 1 above)
Comment (hardiked) [reply packages/app/config/stubs/lru-cache.js]: I am trying to add this file again but then it is failing on pre-commit hook!

**Comment by hardiked** [comment]
Bringing back the lru-cache.js file gives error on pre-commit hook

**Comment by MichaelDeBoey** [comment]
Duplicate of #2729
